### PR TITLE
Charlotte Johansson - portfolio

### PR DESCRIPTION
**Pull request by charlottejohansson**
Still have a lot of styling to do.

https://charlottejohansson-portfolio.netlify.app/

### CODE REVIEW

File: src/components/GlobalStyles.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import styled, { createGlobalStyle } from 'styled-components';
+
+export const GlobalStyles = createGlobalStyle`
```
Comment (JosefineEngdahl) [inline src/components/GlobalStyles.js:3]: Didn't use global styles myself but this was a really nice way of doing it!

File: src/components/FeatureProjects.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import styled from 'styled-components';
+import { Wrapper, SectionHeader } from 'components/GlobalStyles'
+import ProjectList from 'components/ProjectList';
+
+export const FeatureProjects = () => {
+  return (
+    <FeatureWrapper primary>
+      <SectionHeader>FEATURE PROJECTS</SectionHeader>
+      <ProjectList />
```
Comment (JosefineEngdahl) [inline src/components/FeatureProjects.js:10]: I think it's really nice that you kept this component "clean" and import the projectList

File: src/Assets/projects.json [hunk 3]
```diff
@@ -0,0 +1,65 @@
+[
+      {
+        "title": "WEATHER APP",
+        "id": "1",
+        "overlay": "Weather App",
+        "image": "/assetsImg/weatherApp.png",
```
Comment (JosefineEngdahl) [inline src/Assets/projects.json:6]: Your images are really nice, how did you create them?

File: src/Assets/otherProjects.json [hunk 4]
```diff
@@ -0,0 +1,50 @@
+[
```
Comment (JosefineEngdahl) [inline src/Assets/otherProjects.json:1]: I think we got some great inspiration during demon on how to solve the issues with two json-files, but I think this is an okay solution as well :)

**Review by JosefineEngdahl — COMMENTED** [review]
A really nice job Charlotte! Your project-images are amazing! How did you create them?

I think you structured the code and components in a really nice way, it’s easy to follow! Still some work left with the responsiveness, maybe add the icons in the header, and get the "projects-cards” centered, but else it looks great!  ⭐️

File: src/Assets/projects.json (hunk 3 above)
Comment (charlottejohansson) [reply src/Assets/projects.json:6]: Thank you! It's canva, the easiest ever. Start a free account and then search for "tablet" "iphone" etc and choose a template you like. Take a printscreen and just "drag and drop" the pictures in the templates.

Comment (charlottejohansson) [reply src/Assets/projects.json:6]: https://www.canva.com/

**Pull request closed without merging**